rclcpp/CMakeLists.txt

```diff
@@ -48,6 +48,9 @@ macro(target)
   endif()
   add_library(${PROJECT_NAME}${target_suffix} SHARED
     ${${PROJECT_NAME}_SRCS})
+  if("${CMAKE_SYSTEM_NAME} " STREQUAL "WindowsPhone " OR "${CMAKE_SYSTEM_NAME} " STREQUAL "WindowsStore ")
```

Comment:
Since we have moved to CMake 3.5 please avoid the trailing space as well as the quoting and explicit expansion of variables in the condition:

    if(CMAKE_SYSTEM_NAME STREQUAL "WindowsPhone" OR CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")